docs/source/format/CDataInterface.rst

```diff
@@ -79,9 +80,9 @@ Pros of the C data interface vs. the IPC format:
 
 Pros of the IPC format vs. the data interface:
 
-* Works accross processes and machines.
-* Allows data storage and persistency.
-* Being a streamable format, has room for composing more features (such as
+* Works across processes and machines.
+* Allows data storage and persistence.
+* Being a streamable format, the IPC format has room for composing more features (such as
   integrity checks, compression...).
 
 Data type description -- format strings
```

Comment:
Other questions (I can't comment that far outside the diff:

* L322 "If omitted, MUST be 0." That doesn't sound right.
* Should we note below that you can use the ArrowArray struct shape to wrap up RecordBatches and other things?
* "Examples" should probably point to the C++/Python/R stuff we did, right? (If not in this proposal, then later)

Reply:
"If omitted, MUST be 0": I mean that the information is meant to be omitted, then the field should be set to 0. Do you want to propose another wording?

Exporting RecordBatches: yes, I could add a paragraph somewhere later.

I also could add pointers to in-progress implementations somewhere towards the end.

Reply:
I guess what was confusing to me was that setting a field to 0 doesn't sound like omitting it. If I were omitting it, I wouldn't set anything. It sounds like you need to set it always, but if you don't want to enable any of the options, it should be 0--is that right?

Reply:
If you don't set anything, then it could be any junk value, though (C doesn't zero-initialize stuff automatically).

Reply:
So, yes, it needs to be set to the value meaning "omitted" (in R it would be NA? :-)).